src/delivery/templates/kitchen_count.html

```diff
@@ -58,26 +58,30 @@ <h1 class="ui header">{% trans "Kitchen Count Report" %}</h1>
 <table class="ui very compact celled structured table">
   <thead>
    <tr class="top center aligned">
-    <th class="">{% trans 'Clashing' %} - {% trans 'Ingredients' %}</th>
+    <th class="">{% trans 'Clashing' %} {% trans 'ingredients' %}</th>
     <th class="">{% trans 'Qty' %} - {% trans 'Reg' %}</th>
     <th class="">{% trans 'Qty' %} - {% trans 'Lge' %}</th>
     <th class="">{% trans 'Client' %}</th>
-    <th class="">{% trans 'Preparation' %}</th>
     <th class="">{% trans 'Restrictions' %}</th>
    </tr>
   </thead>
   <tbody>
     {% for obj in meal_lines %}
       <tr{% if obj.client == "" and obj.ingr_clash == "" %} class="active"{% endif %}>
-        {% if not obj.span %}
-              <td><strong>{{obj.ingr_clash}}</strong></td>
+        {% if obj.span == "1" %}
+            <td><strong>{% firstof obj.ingr_clash "&nbsp;" %}</strong></td>
         {% elif obj.span != "-1" %}
-              <th rowspan={{obj.span}}>{% firstof obj.ingr_clash "&nbsp;" %}</th>
+              <td rowspan={{obj.span}}>{% firstof obj.ingr_clash "&nbsp;" %}</td>
+        {% endif %}
+        {% if obj.client == "SUBTOTAL" %}
+            <td class="center aligned"><strong>{% firstof obj.rqty "&nbsp;" %}</strong></td>
+            <td class="center aligned"><strong>{% firstof obj.lqty "&nbsp;" %}</strong></td>
+            <td><strong>{% firstof obj.client "&nbsp;" %}</strong></td>
+        {% else %}
+            <td class="center aligned">{% firstof obj.rqty "&nbsp;" %}</td>
+            <td class="center aligned">{% firstof obj.lqty "&nbsp;" %}</td>
+            <td>{% firstof obj.client "&nbsp;" %}</td>
         {% endif %}
-        <td class="center aligned">{% firstof obj.rqty "&nbsp;" %}</td>
-        <td class="center aligned">{% firstof obj.lqty "&nbsp;" %}</td>
-        <td>{% firstof obj.client "&nbsp;" %}</td>
-        <td>{% firstof obj.preparation "&nbsp;" %}</td>
         <td>{{obj.rest_ingr}}{% if obj.rest_ingr %} ; {% endif %} <strong>{% firstof obj.rest_item "&nbsp;" %}</strong></td>
       </tr>
     {% endfor %}
```

Comment:
@manumilou I will make the adjustments according to your review. I will also take into accounts the comments on the labels made by Jean-Francois and Jeremie today.